translations/monero-core_ro.ts

```diff
@@ -983,7 +983,7 @@
     <message>
         <location filename="../pages/Receive.qml" line="521"/>
         <source>&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;This is a simple sales tracker:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Let your customer scan that QR code to make a payment (if that customer has software which supports QR code scanning).&lt;/p&gt;&lt;p&gt;This page will automatically scan the blockchain and the tx pool for incoming transactions using this QR code. If you input an amount, it will also check that incoming transactions total up to that amount.&lt;/p&gt;&lt;p&gt;It&apos;s up to you whether to accept unconfirmed transactions or not. It is likely they&apos;ll be confirmed in short order, but there is still a possibility they might not, so for larger values you may want to wait for one or more confirmation(s).&lt;/p&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>>&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;ACesta este un urmăritor simplu de vânzare:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Permite clientului tău să scaneze acest cod QR pentru a face o plată (dacă acest client are aplicația ce permite scanarea codurilor QR).&lt;/p&gt;&lt;p&gt;Această pagină va scana automat rețeaua pentru a găsi tranzacții inițiate utilizând acest cod QR. Dacă ntroduceți o sumă,  de asemena va verifica tranzacțiile inițiate care insumează acea valoare.&lt;/p&gt;&lt;p&gt;Este la latitudinea ta dacă accepți sau nu tranzacții neconfirmate. Cel mai probabil vor fi confirmate în scurt timp, dar exista o posibilitate ca ele să nu fie confirmate, așadar pentru o sumă mai mare ar fi bine să aștepți una sau mai multe confirmări.&lt;/p&gt;</translation>
```

Comment:
Should be 
` <translation>>&lt;p&gt;&lt;font size=&apos;+2&apos;&gt;Acesta este un urmăritor simplu de vânzare:&lt;/font&gt;&lt;/p&gt;&lt;p&gt;Permite clientului tău să scaneze acest cod QR pentru a face o plată (dacă acest client are aplicația ce permite scanarea codurilor QR).&lt;/p&gt;&lt;p&gt;Această pagină va scana automat rețeaua pentru a găsi tranzacții inițiate utilizând acest cod QR. Dacă introduceți o sumă,  de asemena va verifica tranzacțiile inițiate care însumează acea valoare.&lt;/p&gt;&lt;p&gt;Este la latitudinea ta dacă accepți sau nu tranzacții neconfirmate. Cel mai probabil vor fi confirmate în scurt timp, dar există o posibilitate ca ele să nu fie confirmate, așadar pentru o sumă mai mare ar fi bine să aștepți una sau mai multe confirmări.&lt;/p&gt;</translation>`

(a couple of typos)